arch/lkl/Kconfig

```diff
@@ -42,6 +42,12 @@ config LKL
        select GENERIC_STRNLEN_USER
        select HAVE_ARCH_KASAN
 
+config LKL_FUZZING
+       def_bool y if "$(LKL_FUZZING)" = "1"
```

Comment:
This can be `def_bool n` and then set the kernel config from Makefile.autoconf

Reply:
Fixed.